src/client/src/containers/SelectableCard/index.tsx

```diff
@@ -85,6 +95,20 @@ const SelectableCard = ({
             <CardTitle title={title} />
           </div>
         </div>
+        {isPreview &&
+          isFrameworkSelection &&
+          selected &&
+          (option.internalName === "Flask" ? (
+            <DependencyInfo
```

Comment:
If you are refactoring the other stuff per @piotrmark you could consider something like this here:

```
 <DependencyInfo
    frameworkName={option.internalName}
    installed={dependencies[option.internalName].installed}
     />
```

Reply:
> Can we declare a map instead and then use let's say `let dependencyName = frameworkToDependencyMap[frameworkName]`?

> Same here, a map would be more clear.
> Or maybe even object like:
> `"NodeJS": { "dependencyName": "Node", "downloadLink": "https://nodejs.org/en/download/" }, "Flask": {...}`

(moving @piotrmark comment here since it's hidden)

I will have think a bit more about how this will be refactored (if i want to touch redux store or just include a map from framework -> dependency), but I also don't like how it is right now, maybe a separate PR? And then I will add unit test afterwards

Reply:
@jsondoo why don't you use a selector or something to map the data into a format you want?

Reply:
@SahilTara yea, I will do some refactoring in a separate PR

Reply:
Cool i'm happy with this PR given that you'll be doing the refactorings in a seperate PR. @jsondoo